extension/app/ts/background/settings.ts

```diff
@@ -187,3 +187,15 @@ export async function updateSimulationResults(simulationResults: SimulationResul
 		return await browser.storage.local.set({ simulationResults: SimulationResults.serialize(simulationResults) })
 	})
 }
+
+export async function saveSignerName(signerName: SignerName) {
+	return await browser.storage.local.set({ signerName: signerName})
+}
+
+export async function getSignerName() {
+	const results = await browser.storage.local.get(['signerName'])
+	if (results) {
+		return SignerName.parse(results.signerName)
+	}
```

Comment:
This reinforces my belief (see comment in other PR) that we should either fix the type signature of `browser.storage.local.get` or create a helper (like you did for `set`) that corrects the type signature.
```suggestion
	const results = await browser.storage.local.get(['signerName']) as unknown
	if (typeof results === 'object' && results && 'signerName' in results) {
		return SignerName.parse(results.signerName)
	}
```